src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/CompilerServices/DecimalType.vb

```diff
@@ -0,0 +1,217 @@
+' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System
+Imports System.Globalization
+Imports Microsoft.VisualBasic.CompilerServices.ExceptionUtils
+Imports Microsoft.VisualBasic.CompilerServices.Utils
+
+Namespace Microsoft.VisualBasic.CompilerServices
+
+#Region " BACKWARDS COMPATIBILITY "
+
+    'WARNING WARNING WARNING WARNING WARNING
```

Comment:
I would remove all these compat warning comments and #region markers -- they are 15 years old at this point.

Reply:
Removed